test/integration/http_conn_pool_integration_test.cc

```diff
@@ -162,16 +162,16 @@ TEST_P(HttpConnPoolIntegrationTest, PoolDrainAfterDrainApiAllClusters) {
 
   setUpstreamCount(2);
 
-  auto host = config_helper_.createVirtualHost("cluster_1.com", "/", "cluster_1");
+  auto host = config_helper_.createVirtualHost("cluster_1.lyft.com", "/", "cluster_1");
```

Comment:
ditto about these changes. If they're cosmetic, consider moving these changes to another stand-alone PR.